demos/risk/ScenarioDamage/job.ini

```diff
@@ -15,9 +15,6 @@ asset_hazard_distance = 20
 [fragility]
 fragility_file = structural_fragility_model.xml
 
-[export]
-export_dir = /tmp/
-
```

Comment:
Why do you want to remove the export_dir? The `export_dir` is a good thing, otherwise you will not know where the outputs will be exported when using the ``--exports`` option.